sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -996,15 +997,15 @@ export interface ShareGetPermissionOptions extends CommonOptions {
     abortSignal?: AbortSignalLike;
 }
 
-// Warning: (ae-forgotten-export) The symbol "Permission" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "SharePermission" needs to be exported by the entry point index.d.ts
```

Comment:
![image](https://user-images.githubusercontent.com/10452642/68731155-f6576a00-0583-11ea-980c-05f690a3436e.png)

One live test failed because of the update in content-type `application/json`(unified swagger).
Trying to fix it with a swagger transform.